libs/scripts/scripts.js

```diff
@@ -123,6 +123,7 @@ const config = {
   geoRouting: 'on',
   fallbackRouting: 'on',
   links: 'on',
+  // will undo before merging
```

Comment:
There's still a leftover comment in here

Reply:
Thanks 👍